### DISCUSSION TITLE
Change color of line with conditions

### DISCUSSION THREAD

**Discussion by armeya25** [highfestiva/finplot#451] [General]
Can I do like this,
Say I draw MA50 line and want to change its color of it's below close and above close value,
How to do it,
I already calculated color and saved as series but,
Fplt.add_line((idx, values), color="color_series"))
Doesn't reflect colors

![3ee5cd2c1490e07f780874243b5d1ad511a182fd](https://github.com/highfestiva/finplot/assets/135110431/df74865e-d142-4fa5-b1c2-db5a36b8eb5b)
![f6bce4272881fc79df68b0d90539fde8bc2ea2de](https://github.com/highfestiva/finplot/assets/135110431/77c667b3-ed6d-483f-afb6-ccdc39e78227)

**Comment by highfestiva** [comment]
Hi, see https://github.com/highfestiva/finplot/issues/143 for similar question.

**Reply by armeya25** [reply]
Thank you
